scim2/Package.md

```diff
@@ -27,22 +36,27 @@ import wso2/scim2;
 endpoint scim2:Client scimEP {
     baseUrl:"https://localhost:9443",
     clientConfig:{
-                     auth:{
-                              scheme:"oauth",
-                              accessToken:"<......>",
-                              clientId:"<......>",
-                              clientSecret:"<......>",
-                              refreshToken:"<......>",
-                              refreshUrl:"<......>"
-                          },
-                     targets:[{url:"https://localhost:9443",
-                                  secureSocket:{
-                                                   trustStore:{
-                                                                  filePath:"<......>",
-                                                                  password:"<......>"
-                                                              }
-                                               }
-                              }]
-                 }
+        auth:{
+            scheme:"oauth",
```

Comment:
Can't we define the scheme in the connector itself as it will be always "oauth".

Reply:
SCIM2 specification supports Basic Auth also. Therefore I think the user should be given the choice.